manifests/e/Exploitox/WebRadio/1.0.0.38/Exploitox.WebRadio.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Exploitox.WebRadio
+PackageVersion: 1.0.0.38
+Installers:

```

```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
Installers:
```
This will add the required runtime - if the installer update supersedes this, feel free to resolve, or let me know and I'll resolve.

---

I have now included the runtime in the setup file.